doc/psidk/src/run-infrastructure/administration.md

```diff
@@ -4,20 +4,12 @@ Much of the administration of an individual node can be done via the graphical u
 
 ## Booting a network
 
-This can be done either with the [`psibase`](./cli/psibase.md#boot) CLI tool, or by using the GUI provided by the [admin-sys](../default-apps/admin-sys.md) service. 
-Alternatively, the `POST /native/push_boot` endpoint can be used manually in conjunction with `POST /native/push_transaction` to perform a completely custom boot sequence. See [Understanding boot](#understanding-boot) to learn more.
+Booting a network is only a valid operation if psinode does not yet have any chain. It can be done either with the [`psibase`](./cli/psibase.md#boot) CLI tool, or by using the GUI provided by the [admin-sys](../default-apps/admin-sys.md) service.
+Alternatively, the `POST /native/push_boot` endpoint can be used manually in conjunction with `POST /native/push_transaction` to perform a custom boot sequence. 
 
-### Understanding boot
+The body of the `POST /native/push_boot` request contains a list of transactions that will each be executed in order in the first block. The first transaction is a special transaction known as the genesis transaction. The genesis transaction must use [SystemService::SetCodeSys::setCode] to upload the core services to accounts on the blockchain, and it is not permitted to do anything else.
```

Comment:
That's still not correct. When the genesis transaction is executed SetCodeSys doesn't yet exist. The genesis transaction is fully implemented in native as a single action that takes `GenesisActionData` as its argument.

Reply:
Ahh, right.